1.0.2/build-msbuild/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# TODO pull from microsoft/dotnet when these become available
+FROM microsoft/dotnet-nightly:1.0.2-sdk-msbuild
```

Comment:
So, there won't be a `microsoft/dotnet:1.0.2-sdk-msbuild`? It shows up in the nightly repo. 

> not call it out in the readme

:+1: yeah, we'll try to fix VS templates instead.